tests/test_basic_operation.py

```diff
@@ -53,50 +53,71 @@ def test_005_read_from_secondary():
 def test_006_writes_to_node2_fail():
     node2.run_sql_query("INSERT INTO t1 VALUES (3)")
 
-def test_007_maintenance():
+def test_007_maintenance_primary():
```

Comment:
Maybe good to add a test to show that the `enable_maintenance` fails without `allowFailover`, so with `@raises(Exception)`.